packages/edit-site/src/components/global-styles/typography-panel.js

```diff
@@ -143,10 +147,13 @@ export default function TypographyPanel( { name, element } ) {
 				/>
 			) }
 			{ hasLineHeightEnabled && (
-				<LineHeightControl
-					value={ lineHeight }
-					onChange={ setLineHeight }
-				/>
+				<Spacer marginBottom={ 6 }>
+					<LineHeightControl
+						__nextHasNoMarginBottom={ true }
+						value={ lineHeight }
+						onChange={ setLineHeight }
+					/>
+				</Spacer>
```

Comment:
<img width="150" alt="Typography Panel in Global Styles" src="https://user-images.githubusercontent.com/555336/153934788-6798ed5e-f66a-420e-9a5c-f6802de04cb5.png">

It's confusing because of the naming, but the Typography Panel in Global Styles is actually not a `ToolsPanel`! So it does need some ad hoc spacing. I'm hoping once we remove the outer margins from the rest of the components, we can just wrap them in a single flex/grid gapped container like the actual `ToolsPanel`.

The Storybook one is another issue entirely — I forgot to update it 🙈 Good catch.

Reply:
Sorry, I did confuse this typography-panel.js file with the storybook typography panel that I was testing.

The storybook example for the typography panel was supposed to be a playground for exploring refinements to that panel I believe. If we do not need it perhaps it could be removed. Alternatively, I don't think you need to spend much time on it.

Reply:
> The storybook example for the typography panel was supposed to be a playground for exploring refinements to that panel I believe. If we do not need it perhaps it could be removed.

My thoughts exactly! I'll go ahead and remove it then.